LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2020 Clément Renault
+Copyright (c) 2019-2022 Meili SAS
```

Comment:
```suggestion
Copyright (c) 2018-2022 Meili SAS
```

Reply:
Regarding the cargo toml, seems like it start in 2018 😄

Reply:
Oups, the edition is the rustc edition not the creation date of the repository/crate.